features/topology-map/plugins/org.opennms.features.topology.plugins.topo.linkd/src/main/java/org/opennms/features/topology/plugins/topo/linkd/internal/LinkdTopologyProvider.java

```diff
@@ -38,6 +38,7 @@
 import java.util.Objects;
 import java.util.Set;
 
+import org.apache.commons.lang3.tuple.Pair;
```

Comment:
are we still using apache commons?

Reply:
Apache Commons is fine. We try to use Guava when possible, but there doesn't appear to be a similar class available.

Reply:
I'd like to put al these change in TopologyDao.....